packages/pigz/pigz.spec

```
@@ -0,0 +1,24 @@
+Name: %{_cross_os}pigz
+Version: 2.7
+Release: 1%{?dist}
+Summary: pigz is a parallel implementation of gzip which utilizes multiple cores
+License: zlib
```

```suggestion
License: Zlib AND Apache-2.0
```

This is the SPDX short name for the license. It looks like we also need Apache 2.0 to cover the zopfli sources that are included in the build.